Makefile

```diff
@@ -93,6 +94,11 @@ integration.iid: Dockerfile.integration $(PREFIX)/bin/$(PKG_NAME)_linux-amd64$(c
 test-integration-docker: integration.iid
 	docker run -it --rm $(shell cat $<)
 
+test-plugin:
```

Comment:
I think you want this added to `.PHONY` too.

Reply:
good call. added to .phony